src/broadcast/setup.ts

```diff
@@ -46,6 +92,10 @@ export default function setupBroadcastIpc({
       spectateWorker = await createSpectateWorker(dolphinManager);
     }
     await spectateWorker.connect(authToken);
+
+    // Reset idle timeout on connect
+    await resetSpectateIdleTimeout();
```

Comment:
only reset (implicitly start) spectate idle timeout if we weren't already connected. We can pipe through a return value from `SpectateManager` `connect()` to `SpectateWorker` `connect()` to here to indicate whether we were already connected.